### ISSUE TITLE
Support for sass/scss

### ISSUE THREAD

**Issue by johnsardine** [bingnz/gulp-less-changed#2]
Hi,

Do you think creating a separated version for sass/scss would be too hard? 

This plugin is great and a must-have in my opinion.

I have forked this and am trying to make a sass/scss version. Maybe you could point me in the right direction or add support if is not too hard?

Thanks

**Comment by bingnz** [comment]
Hi John,

Thanks, I'm glad you find the plugin useful! I did consider a sass version but since I wasn't using sass at the time I haven't gotten around to making one. Since this plugin is less-specific at the moment it'd probably make sense to pull out common components into another package and add a small layer on top for less or sass so that the npm modules are separate (gulp-less-changed and a future gulp-sass-changed). Again I'm not really familiar with sass but the [importer callback](https://github.com/medialize/sass.js/blob/master/docs/api.md#the-importer-callback) in the API might be a good starting point. It is something I had on my list to do when I have enough spare time.
Ideally import-lister.ts is the only part that needs to change - I did try and keep the less-specific logic self contained.

David
